flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -550,6 +550,13 @@ class OmpAttributeVisitor : DirectiveAttributeVisitor<llvm::omp::Directive> {
     return false;
   }
 
+  bool Pre(const parser::OmpClause::Uniform &x) {
+    for (const auto &name : x.v) {
+      ResolveOmpName(name, Symbol::Flag::OmpUniform);
```

Comment:
Can you add a `debug-dump-symbols` test to show that the symbol corresponding to name has the `OmpUniform` flag?